src/app_manager/app_manager.py

```diff
@@ -263,6 +264,13 @@ def handle_start_app(self, req):
         if self._current_app:
             if self._current_app_definition.name == req.name:
                 return StartAppResponse(started=True, message="app [%s] already started"%(req.name), namespace=self._app_interface)
+            elif not self._enable_overwrite:
+                return StartAppResponse(
+                    started=False,
+                    message="app [%s] is denied because app [%s] is already running."
+                            % (req.name, self._current_app_definition.name),
+                    namespace=self._app_interface,
+                    error_code=StatusCodes.MULTIAPP_NOT_SUPPORTED)
             else:
                 self.stop_app(self._current_app_definition.name)
             #return StartAppResponse(started=False, message="Please stop the running app before starting another app.", error_code=StatusCodes.MULTIAPP_NOT_SUPPORTED)
```

Comment:
This comment refers to the behavior you implemented above, so you can remove it.

Reply:
I remove the comment, Thank you.